client/app/components/dashboards/AddWidgetDialog.jsx

```diff
@@ -40,14 +34,12 @@ class AddWidgetDialog extends React.Component {
       searchedQueries: [],
       selectedVis: null,
       parameterMappings: [],
-      showModal: false, // show only after recent queries populated to avoid height "jump"
```

Comment:
The visual problem that this fixed will now resurface :/

Reply:
I'll check what I can do here

Reply:
@ranbena I added loader which takes some space, so now user see some feedback on what's going on, and dialog does not "jump" so much.

Reply:
Yeah it looks better 👍